source/exploring_catalog_and_datasets/04_getting_involved/sharing_embedding_data_visualizations.rst

```diff
@@ -0,0 +1,61 @@
+Sharing & embedding data visualizations
+=======================================
+
+Below each visualization are displayed 3 tabs: Share, Embed and Widget. These tabs contain links or pieces of code that allow to share or embed the visualization either elsewhere on the portal or in another website.
+
+.. admonition:: Important
+   :class: important
+
+   All links and codes that can be copied, to either share or embed the visualization, are dynamic. It means that if the visualization is filtered beforehand, the link/code changes to take into account these filters.
+
+
+Sharing a data visualization
+----------------------------
+
+The Share tab below a visualization contains a URL to share a direct access to the visualization.
+
+.. image:: images/share.png
+   :alt: Share tab
+
+Click |icon-copypaste| to copy the URL.
+
+
+Embedding a data visualization
+------------------------------
+
+Data visualizations can either be embedded via an iframe or as a widget. Both are pieces of code that allow anyone to integrate an element directly on a webpage.
+
+An iframe is a way to easily and safely integrate OpenDataSoft content in an isolated frame, independent from the rest of the page. For integrating OpenDataSoft content outside the OpenDataSoft environment, iframes are the best way to go. In terms of interaction and customization, some work would probably be needed.
+Widgets could also be integrated into a page outside the OpenDataSoft environment, but heavy work would be required. They are however ideal for integrating content inside the OpenDataSoft environment. Widgets can also interact with other widgets, which for instance allow the creation of dynamic dashboards using OpenDataSoft.
```

Comment:
It's scary 😄 
We could add a link to the widget documentation ?

Reply:
Agreed for the link to the widget documentation!